tests/wpt-harness/expectations/fetch/api/basic/scheme-blob.sub.any.js.json

```diff
@@ -0,0 +1,56 @@
+{
+  "Fetching [GET] URL.createObjectURL(blob) is OK": {
+    "status": "PASS"
+  },
+  "Fetching [GET] blob:http://www.web-platform.test:8000/ is KO": {
+    "status": "PASS"
+  },
+  "Fetching [POST] URL.createObjectURL(blob) is KO": {
+    "status": "PASS"
+  },
+  "Fetching [OPTIONS] URL.createObjectURL(blob) is KO": {
+    "status": "PASS"
+  },
+  "Fetching [HEAD] URL.createObjectURL(blob) is KO": {
+    "status": "PASS"
+  },
+  "Fetching [PUT] URL.createObjectURL(blob) is KO": {
+    "status": "PASS"
+  },
+  "Fetching [DELETE] URL.createObjectURL(blob) is KO": {
+    "status": "PASS"
+  },
+  "Fetching [INVALID] URL.createObjectURL(blob) is KO": {
+    "status": "PASS"
+  },
+  "Fetching [GET] blob:not-backed-by-a-blob/ is KO": {
+    "status": "PASS"
+  },
+  "Fetching URL.createObjectURL(empty_blob) is OK": {
+    "status": "PASS"
+  },
+  "Fetching URL.createObjectURL(empty_type_blob) is OK": {
+    "status": "PASS"
+  },
+  "Fetching URL.createObjectURL(empty_data_blob) is OK": {
+    "status": "PASS"
+  },
+  "Fetching URL.createObjectURL(invalid_type_blob) is OK": {
+    "status": "FAIL"
```

Comment:
This test checks that blob created with `type: "invalid"` should result in empty `Content-Type` header. It assumes the blob type should me mime parsable although this assumption is not consistent with other blob tests and the spec.

As stated here: https://w3c.github.io/FileAPI/#constructorBlob

> If the [type](https://w3c.github.io/FileAPI/#dfn-BPtype) member of the [options](https://w3c.github.io/FileAPI/#dom-blob-blob-blobparts-options-options) argument is not the empty string, run the following sub-steps:
> - Let t be the [type](https://w3c.github.io/FileAPI/#dfn-BPtype) dictionary member. If t contains any characters outside the range U+0020 to U+007E, then set t to the empty string and return from these substeps.
> - Convert every character in t to [ASCII lowercase](https://infra.spec.whatwg.org/#ascii-lowercase).